test/testdata/deployednettemplates/recipes/fastnet/genesis.json

```diff
@@ -0,0 +1,104 @@
+{
+	"NetworkName": "",
+	"VersionModifier": "",
+	"ConsensusProtocol": "future",
```

Comment:
We may want to change this to alpha1

Reply:
yes, that version is defined in https://github.com/algorand/go-algorand/tree/feature/alphanet

Reply:
Updated to alpha1.